### DISCUSSION TITLE
Babel plugin

### DISCUSSION THREAD

**Discussion by toggery** [cawa-93/vite-electron-builder#173] [Q&A] [answered]
How to use `babel-plugin-import`? I want to use some UI component e.g. antd.

**Comment by cawa-93** [comment, marked as answer]
https://github.com/0ahz/vite-plugin-babel-import

**Reply by toggery** [reply]
Great! It run as expected in `development` mode. Unfortunately, the compiled electron app run **without css styles**.

I tried another plugin https://github.com/onebay/vite-plugin-imp, it run as expected in both modes, but the command line displayed as follows:
```sh
Sourcemap is likely to be incorrect: a plugin (vite-plugin-imp) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
```

**Reply by cawa-93** [reply]
You should write to the developers of these plugins.

**Reply by toggery** [reply]
Thanks.
